### PR TITLE
Metric-Hub integration

### PR DESCRIPTION
**Pull request by ANich**
Initial metric-hub integration - syncs to business glossary.

### CODE REVIEW

File: sync/metrichub.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import re
+from dataclasses import dataclass
+from typing import List
+
+import sqlglot as sqlglot
+from metric_config_parser.config import ConfigCollection
+
+METRIC_HUB_REPO_URL = "https://github.com/mozilla/metric-hub"
+METRIC_HUB_DOCS_URL = "https://mozilla.github.io/metric-hub"
```
Comment (scholtzan) [inline sync/metrichub.py:9]: The goals is to eventually get rid of this and use datahub as place for the docs instead.

File: sync/metrichub.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    config_collection = ConfigCollection.from_github_repo(METRIC_HUB_REPO_URL)
+
+    metrics = []
+    for definition in config_collection.definitions:
+        for (
+            metric_name,
+            metric,
+        ) in definition.spec.metrics.definitions.items():
+            # TODO: Some metrics don't have data sources, (e.g. ad_click_rate, a chained metric used in jetstream)
+            #  Should we show these in the catalog?
```
Comment (scholtzan) [inline sync/metrichub.py:97]: yeah, we should still show them

File: sync/metrichub.py (hunk 1 above)
Comment (ANich) [reply sync/metrichub.py:9]: 👍 Makes sense, I'll link to the GitHub repo directly then

**Review by Iinh — APPROVED** [review]
LGTM, thanks!

**Pull request merged**